.circleci/config.yml

```diff
@@ -194,14 +196,15 @@ workflows:
   version: 2
   test-suite:
     jobs:
-      # this is now a slow process... let's only run on master
+      # this is now a slow process... let's only run on main and PR branches we care about
```

Comment:
The `docker-image` job is not blocking the CI completion. So I vote for removing the filter here (only keep the filter on the publishing step). Is this okay for you, @alpe?
<img width="1058" alt="Bildschirmfoto 2022-09-20 um 10 15 47" src="https://user-images.githubusercontent.com/2603011/191205291-57489a4c-05e8-46d0-953b-3e5cefbdc65a.png">

Reply:
okay, it's not even close
<img width="723" alt="Bildschirmfoto 2022-09-20 um 10 26 43" src="https://user-images.githubusercontent.com/2603011/191207670-50326091-5651-4f0c-8592-494c2d359405.png">